### ISSUE TITLE
User and password are not filled in on Jellyfin

### ISSUE THREAD

**Issue by stroobandt** [alandmoore/admbrowser#20]
The user and password are not automatically filled in —like other browsers with password management do— on the [Jellyfin](https://Jellfin.org) media system.

This can be tested with the following demo YAML file:

```yaml
default_password: ''
default_user: demo
navigation: False
start_url: https://demo.jellyfin.org/stable/web/index.html
window_size: max
```

This would be nice to have for a kiosk browser without keyboard.

**Comment by alandmoore** [comment]
The default_user and default_password settings only apply to sites that request HTTP authentication, not those that handle authentication at the application level.  

Implementing a password manager would be nice, but is well beyond the scope of this particular feature.  I think the documentation should perhaps be updated to better explain the purpose of those configuration options.

**Comment by stroobandt** [comment]
Yes, Alan, from the many hours I spent studying at Hull University's library (the workplace of British poet Philip Larkin) it dawned on me that this feature may be solely related to HTTP authentication. :-)

In the end, I solved my problem by setting `privacy_mode: False` and now the Jellyfin media server recognises me again without asking for any authentication. This issue can therefore be closed.
